snippets/nordic-ppr-ram/nordic-ppr-ram.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* code partition size must match RAM size */
+&cpuppr_code_partition {
+	reg = <0x126000 DT_SIZE_K(28)>;
+};
```

Comment:
This adjustment isn't required when building for PPR, only for its owner.

Reply:
good catch, actually dropped the snipped as it was just CONFIG_XIP setting...